SMCollection/schema-LD.json

```diff
@@ -0,0 +1,60 @@
+{

```

Comment:
bad file naming. see the previous comment regarding this point.
Why schema.json and schema-LD. json? (They are for validating key-values payloads)

Reply:
After conversations with Ahmed, I understand that sometimes it's necessary to use both schema.json and schema-LD.json. Schema.json and example.json files will be used for key-values payloads and schema-LD.json and example-normalized.jsonld files will be used for more complex payloads. Please offer your advice as of course I might have understood poorly.

Reply:
This feature (having a specific model for example-normalized.jsonld data models is under assessment), and therefore it is not currently required). Not in [contribution manual](https://bit.ly/contribution_manual).